guardian/media-atom-maker
# Upgrade google dependencies

## What does this change?

This PR updates various google dependancies in an effort to help resolve issues that users are currently experiencing.

This project includes an unmanaged dependency, the YouTube Content ID API library. The latest version of this can be found at https://developers.google.com/youtube/partner/client_libraries

## How to test

The unit tests for this build pass. We should deploy the changes to CODE to ensure no regressions are introduced. 

## How can we measure success?

We can eliminate or rule out, API changes as the cause of the recent user issues.

## Have we considered potential risks?

The changes to these dependencies could cause unexpected issues.
